conf/config.neon

```diff
@@ -230,7 +230,7 @@ parameters:
 	errorFormat: null
 	pro:
 		dnsServers:
-			- '1.1.1.1'
```

Comment:
Would it make sense to use both?

Reply:
Probably not. I don't know exactly how the React DNS Config object works but if it would randomly pick one of the servers then there's 50% chance it would pick the one that doesn't work, making debugging it a bit harder because it would sometimes work.

If it would try one and then the other one if it fails, then it would possibly introduce a delay.